packages/manager/modules/pci/src/projects/project/ai/apps/add/add.controller.js

```diff
@@ -323,9 +327,24 @@ export default class AppAddController {
     const lastImagePart = splitImage[splitImage.length - 1];
     const [prefix] = lastImagePart.split(':');
     this.appModel.name = `${prefix}-${nameGenerator()}`;
+    this.appModel.command = this.commandLine
+      ? this.splitIntoArray(this.commandLine)
+      : [];
```

Comment:
```suggestion
    this.appModel.command = this.splitIntoArray(this.commandLine);
```
The test you're making is also done in the splitIntoArray method, so no need to do it again here